criu/util.c

```diff
@@ -878,6 +878,8 @@ void split(char *str, char token, char ***out, int *n)
 		cur++;
 	}
 
+	if (*n == 0)
```

Comment:
*out = NULL?

Reply:
it is possible only if str is null, I am not sure that we want to handle this case. But if we want, we need to add a comment here.

Reply:
I added a comment. Let me know if I should just drop it.